### ISSUE TITLE
A new naval branch: Hovercrafts

### ISSUE THREAD

**Issue by jaygamer81** [Anuken/Mindustry-Suggestions#1300]
**Describe the content or mechanics you are proposing.**
Tldr (summary); a new naval branch with cool properties such as being able to fly over land and being able to submerge in water so it cannot attack nor can anything else really attack it. It would also be able to store units and lots of items. It would be a multi purpose unit with many uses and quirks. They would be mostly weak to air units and other naval units and walls. I think this should be added in V7 perhaps as it is alot of content and may require more counters. Read the entire thing for more detail.

I think there should be a hovercraft naval branch, a branch which would be worse than naval or air branches health and damage wise but would be sleek and mobile. There would also be certain tiers which can "submerge", submerging would allow the unit to be undetectable by units or turrets until they reach shallow water or they hit a wall; submerged units cannot attack. They would have the ability to fly over walls while being able to be a naval unit when on water; like the support ground unit branch in a way. They would cost 25 meta-glass, 35 silicon and 15 titanium when being made. The T1 unit called glide (all names are work in progress), would be the first hovercraft; it would be armed with a salvo-like gun which would deal the same damage as a salvo firing copper, it wouldn't be able to submerge and would have only 245 health. The T2 unit would be called Oasis; it would have the ability to submerge and would fire a swarm-type rocket launcher which would be surge-alloy based (they have arcs when exploding), this unit would have 300hp; it cannot hover overland. The T3 unit know as tide would have 2-hail class artillery turrets and two duo turrets with the ability to hover it can also drop bombs (the same as the horizon bombs) when flying over land; it cannot submerge in water and would have 500 health; it can also hold units. The T4 unit known as Neptune would have two scatter like turrets with a double barreled ripple; it would also have the ability to submerge, but it cannot hover; it would also be armed with **torpedos** which would essentially only fire when non-submerged; they would be really slow with crippling damage of 800 per torpedo; the con is that this can only attack buildings on shallow water or naval units that are submerged or above water, it would also have a radar like effect which will make it target submerged units; this unit has 5000hp, it can also hold units. The last unit would be Poseidon, a massive ship with the ability to both fly and submerge, this unit can detect submerged units for all nearby units or turrets to target, it can also boost nearby units with an overdrive projector when not submerged, it is armed with two long ranged cannons which would fire up to 65 blocks away; they would deal 300 damage per shell; this unit has 12,500 hp and can hold alot of units in its cargo, it can also hold air units too, it can also hold up to 1000 items in it, making it good for transport.

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**
This would add a second naval unit branch with new properties. It would also add a new weapon with new tactics required to kill these units. This would also be a really unique branch as it can be in air, water and its new status, submerged. It is not op or too weak. 
The counter for the T1 would be almost everything, the counter for the T2 would be walls and everything. T3 would be weak to air units as it only has duos to combat. T4 and T5 would be weak against air units also.

**Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** *Failure to do so may result in your request being closed automatically.*

1. - [X] I have done a quick search in the list of suggestions to make sure this has not been suggested yet.
2. - [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
3. - [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
4. - [X] I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by scarf-generic** [comment]
really overpowered submarines. take #69 or #690, make them able fly and you have this suggestion.

**Comment by Bohaska** [comment]
https://github.com/Anuken/Mindustry-Suggestions/issues/690 / https://github.com/Anuken/Mindustry-Suggestions/issues/69 = https://github.com/Anuken/Mindustry-Suggestions/issues/10

**Comment by jaygamer81** [comment]
> really overpowered submarines. take #69 or #690, make them able fly and you have this suggestion.

It isnt overpowered. Just get some air units or some other T4/T5 hovercrafts and you can rat them out. Also just make a massive wall blocking them at a river delta; then they will have to surface to destroy the wall and continue, which may even kill them since they are a quite underpowerfull.

**Comment by Kyllingene** [comment]
While this is indeed support naval, it is not quite the same as the main point is not support, but multifunctionality.
